edit/linter.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var linter = (() => { // eslint-disable-line no-var
+  const changeCallbacks = [];
+  const unhookCallbacks = [];
+  const linters = [];
+  const cms = new Set();
+
+  return {
+    register,
+    refresh,
+    hook,
+    unhook,
+    onChange,
+    onUnhook
+  };
+
+  function onUnhook(cb) {
+    unhookCallbacks.push(cb);
+  }
+
+  function onChange(cb) {
```

Comment:
Or `onUpdated`?

Reply:
this would also sound like the linter itself is updated

Reply:
`onFinished`?

Reply:
My reasoning was, since these are invoked from onUpdateLinting, the event name should reflect that because it'll be obvious and won't add a new unexpected term. We can use the full name e.g. onLintingUpdated if you really want a dedicated event.